charts/memgraph-high-availability/values.yaml

```diff
@@ -62,6 +62,12 @@ sysctlInitContainer:
   enabled: true
   maxMapCount: 262144
 
+secrets:
```

Comment:
This needs to end up in the README describing the property: https://github.com/memgraph/helm-charts/blob/main/charts/memgraph-high-availability/README.md#configuration-options

Reply:
This also needs to be added to the docs page, [something like this on a HA part of the page](https://memgraph.com/docs/getting-started/install-memgraph/kubernetes#secrets)  also [here](https://memgraph.com/docs/getting-started/install-memgraph/kubernetes#configuration-options-1)

Reply:
Can you just confirm you have been able to setup the secrets inside the cluster and connect to Memgraph instances with auth? 

From the code perspective it looks ok, but our CI/CD is currently not operational for HA so I want to be sure this works.

Reply:
Yes, I have now tested and it works.

Reply:
This is resolved.